server/ansible/deploy.yaml

```diff
@@ -0,0 +1,44 @@
+- name: Deploy updates to Draft
+  hosts: all
+  tasks:
+
+    - name: Pull new commits from GitHub
+      git:
+        repo: 'https://github.com/private-attribution/draft.git'
+        dest: '{{ ansible_env.HOME }}/draft'
+        update: yes
+
+    - name: Install packages based on package-lock.json via npm
+      npm:
+        path: '{{ ansible_env.HOME}}/draft/server'
+        state: present
+        ci: true
+
+    - name: Load .env file
```

Comment:
Can you please help me understand what is getting stored in `.env` file here? According to [cli doc](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/secretsmanager/get-secret-value.html#examples)

> The decrypted secret value, if the secret value was originally provided as a string or through the Secrets Manager console.

> If this secret was created by using the console, then Secrets Manager stores the information as a JSON structure of key/value pairs.
 
If that's true, I am a little nervous about keeping the secret on durable storage. Is it possible to configure env variable or (probably better) assume role on this instance?

Reply:
It's variables needed server side like API keys, db passwords, etc. We could certainly set these as environmental variables instead of using the `.env` file, and it seems reasonable to avoid writing them to a file.

When you say "assume role on this instance`, this does assume a specific AMI role, which is what allows the `aws secretmanager` cli to actually access those secrets. Is there something else you mean by this?

Reply:
ok, fixed. I added a shell script "load_secrets.sh" which is sourced and adds them into the env. the TLS keys are still written to disk, because traefik expects them there. it's not immediately obvious of a better way to handle that.

Reply:
thank you! 

> When you say "assume role on this instance, this does assume a specific AMI role, which is what allows the aws secretmanager` cli to actually access those secrets. Is there something else you mean by this?

Yea I was thinking about using https://awscli.amazonaws.com/v2/documentation/api/latest/reference/sts/assume-role.html to get temporary credentials, but this serves different purpose as far as I can tell.

If we store more than one password, API key, etc, we could consider partitioning this and store one secret with one key